sage_patchbot/patchbot.py

```diff
@@ -1438,8 +1438,7 @@ def good(report) -> bool:
                 print("\n\n")
                 print("Current base: {} {}".format(patchbot.config['base_repo'],
                                                    patchbot.config['base_branch']))
-                print("Failing tests in your base install: exiting.")
-                sys.exit(1)
+                print("Warning: Failing tests in your base install.")
```

Comment:
I do not agree with that. The patchbot must run on a sane ground, and must refuse to run if base has failing doctests. If you want to avoid that, use the existing option "skip-base".

Reply:
Would you care to review https://trac.sagemath.org/ticket/33233, which would enable implementing a more flexible solution?